phpstan.neon

```diff
@@ -5,13 +5,14 @@ includes:
     - vendor/phpstan/phpstan-phpunit/extension.neon
     - phpstan-extension.neon
 
-parameters:
-    level: max
+conditionalTags:
+    PhpParser\NodeVisitor\NodeConnectingVisitor:
+        phpstan.parser.richParserNodeVisitor: true
 
-    featureToggles:
-        finalByPhpDocTag: true
+parameters:
+    level: 8
```

Comment:
In `phpstan-default.neon` we still have `max`. How can we expect max level from others but hold ourselves just at 8?

Reply:
Also, this is rather large change, moving from max down to 8 - should be described in the PR desc.

Reply:
...oh, the PR desc is completely empty... That should be fixed.